privaterelay/views.py

```diff
@@ -139,11 +140,9 @@ def _get_event_keys_from_jwt(authentic_jwt):
 
 
 def _handle_fxa_profile_change(authentic_jwt, social_account, event_key):
-    token = social_account.socialtoken_set.first()
-    headers = {'Authorization': 'Bearer: {0}'.format(token.token)}
-    resp = requests.get(
-        FirefoxAccountsOAuth2Adapter.profile_url, headers=headers
-    )
+    client = _get_oauth2_session(social_account)
+    resp = client.get(FirefoxAccountsOAuth2Adapter.profile_url)
```

Comment:
Do we not need `headers=headers` on the GET? Or is it handled by the new client being created using `_get_oauth2_session` magically 🧙?

Reply:
The new client should be handling that part now.